pkgpanda/build/__init__.py

```diff
@@ -1230,7 +1294,10 @@ def clean():
     # Bundle the artifacts into the pkgpanda package
     tmp_name = pkg_path + "-tmp.tar.xz"
     make_tar(tmp_name, cache_abs("result"))
-    os.rename(tmp_name, pkg_path)
+    if is_windows:
+        os.replace(tmp_name, pkg_path)
+    else:
+        os.rename(tmp_name, pkg_path)
```

Comment:
We should be able to use `os.replace()` on all platforms.

Reply:
@paulcallen -- fix